packages/tc-api-db-manager/index.js

```diff
@@ -23,15 +23,13 @@ const initConstraints = async () => {
 		const retrieveConstraints = async () => {
 			const constraints = await executeQuery('CALL db.constraints');
 			return constraints.records.map(constraint =>
-				constraint.get('description'),
+				constraint.get('name'),
```

Comment:
This, and the similar line below, seem to be the solution to the original failure of the db to start. The reasons being that:
a) the description field is no longer available
b) the db.constraints and db.indexes procedures have been deprecated